### DISCUSSION TITLE
how to move vertical zero.

### DISCUSSION THREAD

**Discussion by dkebler** [OpenHantek/OpenHantek6022#255] [Q&A] [answered]
Running latest stable on linux.  with just the ch1 trace The zero(v) trace appears (vertically) 3/4 up instead of  in the middle of the display like on the example in the manual.  I see no spinner in the UI to set the vertical position of zero(v).  

I'm an oscilloscope noob but just seems like I should be able to set the vertical zero position like for example I have only a positive voltage trace so no need to see the bottom half so move the zero to the bottom and then I can possibly use a smaller scale for more detail.

merry xmas, trying out my new toy amazon santa brought me all the way from china.

from manuel
![image](https://user-images.githubusercontent.com/1316185/147391961-08022326-0af8-48e3-89f4-d40a82488528.png)

on my screen
![image](https://user-images.githubusercontent.com/1316185/147391922-ae483ea8-a32b-4a3a-aede-a21d98a21e52.png)

**Comment by Ho-Ro** [comment, marked as answer]
Grab the CH1 (or CH2) on the left and drag it up/down.
![image](https://user-images.githubusercontent.com/12542359/147392325-ea5c1a5a-7237-4c32-87bc-5365dc3b1f81.png)
The same applies to the trigger level on the right side and the trigger position on the top.
